components/inlineaiprompt/overview.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Overview
+page_title: InlineAIPrompt Overview
+description: Overview of the InlineAIPrompt for Blazor.
+slug: inlineaiprompt-overview
+tags: telerik,blazor,inlineaiprompt,overview
+published: True
+position: 0
+---
+
+# Blazor InlineAIPrompt Overview
+
+The UI for Blazor InlineAIPrompt is a popup-based component that lets you interact with AI language models right inside your content.
+
+It provides a simple and focused way to send prompts and get responses from AI without interrupting the user’s flow. The InlineAIPrompt is great for adding contextual AI help exactly where users need it.
+
+## Creating Blazor InlineAIPrompt
+
+1. Add the `<TelerikInlineAIPrompt>` tag.
+2. Subscribe to the `OnPromptRequest` event that will fire whenever the user sends a prompt request. The handler expects an argument of type `InlineAIPromptPromptRequestEventArgs`.
+3. Set the `Prompt` parameter
+4. (optional) Set the `PromptContext` parameter
+
+>caption Telerik Blazor InlineAIPrompt
+
+````Razor
+<div class="products-container">
+    @foreach (var product in Products)
+    {
+        <div class="product-card"
+             @onclick="@((MouseEventArgs e) => ShowPromptAsync(e, product))"
+             @onclick:preventDefault="true">
+            @product
+        </div>
+    }
+</div>
```

Comment:
Consider three TelerikButtons for simplicity and shorter code. Applies to all examples.